### ISSUE TITLE
json-schema to suretype: import aliases are empty

### ISSUE THREAD

**Issue by gCardinal** [grantila/typeconv#18]
Running a simple conversion from json-schema to suretype results in imports from suretype that are unusable. See below:

```typescript
/* tslint:disable */
/* eslint-disable */
/**
 * This file is generated by core-types-suretype on behalf of typeconv, DO NOT EDIT.
 * For more information, see:
 *  - {@link https://github.com/grantila/core-types-suretype}
 *  - {@link https://github.com/grantila/typeconv}
 */

import { suretype as , v as , compile as , annotate as  } from 'suretype';

// ... rest of the schemas, interfaces and validators
```

Generation was done in the most basic way possible I believe:

```js
const fs = require('fs');
const path = require('path');
const typeconv = require('typeconv');

const { readdirSync, readFileSync, writeFileSync } = fs;
const { resolve, extname, basename } = path;
const { getJsonSchemaReader, getSureTypeWriter, makeConverter } = typeconv;

const schemasDirectory = resolve(__dirname, 'schemas');
const { convert } = makeConverter(getJsonSchemaReader(), getSureTypeWriter())

readdirSync(schemasDirectory).forEach(file => {
    if (extname(file).includes('json')) {
        const fileName = basename(file, extname(file));
        const filePath = resolve(__dirname, 'schemas', file);
        const fileContent = readFileSync(filePath).toString();
        const targetFile = resolve(__dirname, 'schemas', `${fileName}.ts`);

        convert({ data: fileContent }).then((converted) => {
            writeFileSync(targetFile, converted.data)
        })
    }
});

```

**Comment by EricCrosson** [comment]
I am seeing this too, using the following command

```
typeconv --from-type ts --to-type st -o suretype ./interfaces.ts
```

with typeconv 1.7.0

**Comment by EricCrosson** [comment]
ah, but only when `typeconv` was installed globally! When I used typeconv installed locally to my package.json, this error did not reproduce

**Comment by moltar** [comment]
I'm seeing the same, regardless of the installed location.

Running as:

```sh
node_modules/.bin/typeconv --from-type=oapi --to-type=st --output-directory=src/ foo.json
```

**Comment by aphex** [comment]
Same issue here, just doing string replacement for now to clean it up but would be nice to get this fixed.

**Comment by grantila** [comment]
Fixed in 1.8. This was caused by TypeScript breaking the API/ABI in a minor version, so having TypeScript > 4.5 installed in the same repo broke this.
Now depending on 4.7.
